### PR TITLE
[Components with more than 1 service] LPS-175994 Refactor DTO Converter components with more than 1 service type from Headless

### PR DESCRIPTION
**Pull request by MrJohn1911**
Related pull: https://github.com/liferay-core-infra/liferay-portal/pull/3551

In the previous pull, there was no regressions found, I did this one just to organize the commits, so I think it's good to go.

### CODE REVIEW

**Comment by MrJohn1911** [comment]
ci:test:headless

**Comment by MrJohn1911** [comment]
ci:test:startup

File: modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/main/java/com/liferay/headless/admin/user/internal/dto/v1_0/util/DTOConverterUtil.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.headless.admin.user.internal.dto.v1_0.util;
+
```
Comment (rafaprax) [inline modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/main/java/com/liferay/headless/admin/user/internal/dto/v1_0/util/DTOConverterUtil.java:16]: Hey @MrJohn1911, please move it to the com.liferay.headless.admin.user.internal.dto.v1_0.converter.util package

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Review by rafaprax — APPROVED** [review]

**Comment by MrJohn1911** [comment]
ci:test:sf

**Comment by MrJohn1911** [comment]
ci:test:relevant

**Comment by MrJohn1911** [comment]
ci:test:startup

**Comment by MrJohn1911** [comment]
ci:test:headless

**Comment by zhangyan0701** [comment]
No regressions found.

**Pull request closed without merging**